ckan/adx_config.ini

```diff
@@ -125,7 +127,6 @@ ckanext.validation.run_on_update_async = True
 ckanext.validation.allow_invalid_data = True
 ckanext.validation.default_validation_options = {"row_limit": 20000, "error_limit": 100, "skip_checks": ["extra-header"] }
 ckanext.validation.formats = csv xlsx xls shp geojson
-ckanext.validation.allow_invalid_data = True
```

Comment:
Your reasoning is that this doesn't matter any more because we have gone to async?

Reply:
I haven't thought this through properly, just wanting to check you have...

Reply:
It's just a duplicate line (from 3 above.